### PR TITLE
Intial setup

### PR DESCRIPTION
**Pull request by sohan9819**
Added pages and routes 

image

### CODE REVIEW

**Review by rahul31agrawal — APPROVED** [review]
Looks good to merge

**Comment by athar05** [comment]
The UI looks good. I especially liked the transition between login form and signup form. Just amazing. 

One thing i noticed, when clicking on "Back to home" from the "/auth" page, it takes the user to "/" page, when ideally it should take the user to "/home" page.

Rest looks good. Looking forward to seeing how this app comes out.

File: src/components/Nav.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          onClick={() => {
+            navigate('/auth');
+          }}
+          className='nav__login__button'
+        >
+          <i className='bx bx-log-in'></i> Login
+        </button>
+        {/* <button className='nav__login__button'>
+          <i className='bx bx-log-out'></i> Logout
+        </button> */}
```
Comment (teja538) [inline src/components/Nav.jsx:45]: can remove commented code

**Review by teja538 — COMMENTED** [review]
remove the commented code
everything looks good
good to see that you used variables and readable naming convensions

**Pull request merged**